proto/one/enterprise/data/post_filtering.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+import "post_filtering_column_definition.proto";
+import "google/protobuf/wrappers.proto";
+
+option csharp_namespace ="ONE.Models.CSharp.Enterprise.Data";
```

Comment:
Currently all of our models are in the ONE.Models.CSharp namespace. I would prefer not to add more namespaces unless you see a benefit to doing so.

Reply:
The ONE.Models.CSharp.Enterprise.Data namespace is used by the EnterpriseDatasetDefinition protobuf which defines version 0 and 1 of the report definition json. This new code adds support for the report definition json version 2 so I added it to the same namespace.

Reply:
Ah, I did not realize we had protobufs in multiple namespaces.  Makes sense to use the same one I guess.